Instructions on how to install binaryen (#94)
paritytech/ink-docs: docs/getting-started/setup.md

As a pre-requisite for the tool you need to install the [binaryen](https://github.com/WebAssembly/binaryen) package, which is used to optimize the WebAssembly bytecode of the contract.

> For example, on Ubuntu or Debian, run the following command:

Reviewer: ```suggestion
To install it on Ubuntu or Debian, run the following command:
```

```bash
sudo apt install binaryen

> ```

Reviewer: The latest stable [binaryen release is `v99`](https://tracker.debian.org/pkg/binaryen) which is too old for us to use, so this actually doesn't work at the moment. However, it is still a valid way of installing `wasm-opt`, so we can keep it but can you add a [caution admonition](https://docusaurus.io/docs/markdown-features/admonitions
) right below the installation instruction mentioning that users should manually download an up to date binary instead?

> For macOs run
> ```bash
> brew install binaryen
> ```
> For other operating systems, download from [WebAssembly releases](https://github.com/WebAssembly/binaryen/releases).

Reviewer: ```suggestion

For installation on macOS run:

```bash
brew install binaryen
```

For other operating systems, download from [WebAssembly releases](https://github.com/WebAssembly/binaryen/releases).
```
